test/karma.conf.js

```diff
@@ -18,7 +18,10 @@ module.exports = function (config) {
         },
         browserify: {
             debug: true,
-            transform: [['babelify', { 'presets': ['es2015'] }]]
+            transform: [['babelify', {
+                "presets": ["@babel/preset-env"],
```

Comment:
good point! they are not equal. the changes should not go live until this is not fixed

Reply:
transform synced with our other components babel transform setup